lib/renderer.js

```diff
@@ -47,7 +47,7 @@ default_rules.fence = function (tokens, idx, options, env, slf) {
   }
 
   if (options.highlight) {
-    highlighted = options.highlight(token.content, langName) || escapeHtml(token.content);
+    highlighted = options.highlight(token.content, langName, token.attrs) || escapeHtml(token.content);
```

Comment:
Where do you get `token.attrs` from? Shouldn't it be second half of `token.info`?